spec/std/file_spec.cr

```diff
@@ -507,108 +537,117 @@ describe "File" do
     end
   end
 
-  describe "expand_path" do
-    it "converts a pathname to an absolute pathname" do
-      File.expand_path("").should eq(base)
-      File.expand_path("a").should eq(File.join([base, "a"]))
-      File.expand_path("a", nil).should eq(File.join([base, "a"]))
-    end
-
-    it "converts a pathname to an absolute pathname, Ruby-Talk:18512" do
-      File.expand_path(".a").should eq(File.join([base, ".a"]))
-      File.expand_path("..a").should eq(File.join([base, "..a"]))
-      File.expand_path("a../b").should eq(File.join([base, "a../b"]))
-    end
+  {% unless flag?(:win32) %}
+    describe "expand_path" do
```

Comment:
`expand_path` is utterly broken beyond repair on windows.

Reply:
would be solved by #5550

Reply:
Indeed. That's why I'm pushing for it to be merged.